### ISSUE TITLE
Response serialization failed

### ISSUE THREAD

**Issue by LiroyvH** [NoahFetz/F1AppleTV#67]
When trying to play the Bahrain in 30-minutes recap video, the app throws an error "Response serialization failed: response could not be decoded because of error 'the data could not be read because they're missing'. And then some more output. See screenshot. 

![image](https://user-images.githubusercontent.com/4643445/224872089-fa8c2f8e-116e-4180-b20c-3969f4a25388.jpeg)

Warm regards

**Comment by NoahFetz** [comment]
Looks like you're not authorized to play this stream

**Comment by LiroyvH** [comment]
@NoahFetz Yeah it's odd, even weirder that one of the two works and the other gives an error, heh!

**Comment by ppodevlab** [comment]
Hi. Im testing the app today with the race and getting similar error

![Uploading 7826FA81-0C12-44C2-BDCC-1019DACE1314.jpeg…]()

**Comment by ppodevlab** [comment]
Im testing the app today and getting similar error.

![C50426EF-FCA3-4325-9200-F615AC7E7C8D](https://user-images.githubusercontent.com/7081272/226192500-0f2c44f9-6134-4507-9813-8e7fcc293e0b.jpeg)

**Comment by NoahFetz** [comment]
This means you're not authorized for this stream, maybe your region isn't supported

**Comment by erikvanaerle** [comment]
> This means you're not authorized for this stream, maybe your region isn't supported

Same error here (region: NL) when trying to log in to the F1 account🙁

**Comment by vinodhmoodley** [comment]
I get the same error if I try to login. The official app works but the quality keeps fluctuating. Vroom TV is perfectly stable in comparison but now I can’t login anymore. I’m in South Africa and it’s a supported region.

**Comment by NoahFetz** [comment]
> I get the same error if I try to login. The official app works but the quality keeps fluctuating. Vroom TV is perfectly stable in comparison but now I can’t login anymore. I’m in South Africa and it’s a supported region.

You probably hit the device limit. You need to remove one from the web portal

**Comment by vinodhmoodley** [comment]
I didn’t even know there was a device limit. Works fine now. Thank you.
